File: Labs/Lab 1/webUI-test/src/test/java/com/accessHQ/models/Nav.java
package com.accessHQ.models;

import org.openqa.selenium.By;
import org.openqa.selenium.chrome.ChromeDriver;

public class Nav {
    private final ChromeDriver driver;
    private final String tab;

    public Nav(ChromeDriver driver, String tab) {
        this.driver = driver;
        this.tab = tab;
    }

    public void clickTab(){
        driver.findElement(By.cssSelector("[aria-label = planets]")).click();
    }
}

File: Labs/Lab 2/AccessHQ.NETTestProject/Models/Nav.cs
using OpenQA.Selenium;

namespace AccessHQ.NETTestProject.Models
{
    internal class Nav
    {
        private WebDriver driver;

        public Nav(WebDriver driver)
        {
            this.driver = driver;
        }

        public void NavigateToPlanets()
        {
            driver.FindElement(By.CssSelector("a[href='#/planets']")).Click();
        }
    }
}
